### DISCUSSION TITLE
Permissions in Snipe-IT

### DISCUSSION THREAD

**Discussion by eliecerserna** [snipe/snipe-it#11993] [General]
Does anyone know how I can give permissions to a user so that they only see the devices of one company?

I have the inventory of two different companies, I want to be able to create a user for each company and that he only sees the devices that belong to his company, I have played with the permissions but the two users continue to see all the devices.

**Comment by peter-goedtkindt-IMTF** [comment]
Good point - permissions should be based on "organisational units"  - with maybe some users having the full overview, and others only able to act on their department, or the mobile phones of that department.
